modules/perception/camera/tools/offline/visualizer.h

```diff
@@ -52,8 +53,15 @@ class Visualizer {
   std::string type_to_string(const apollo::perception::base::ObjectType type);
   std::string sub_type_to_string(
       const apollo::perception::base::ObjectSubType type);
-
+  Eigen::Matrix3d homography_im2car() {
+      return homography_im2car_;}
```

Comment:
Run "scripts/clang-format.sh modules/perception" if you are not sure about the correct code layout.
It will show you how to fix most of issues in the pull request that I didn't point out :)

Reply:
I've run the script to apply all the style-related changes